static/js/publisher/tour.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import Tour from "./tour/tour";
+
+export default function initTour() {
+  const tour = document.createElement("div");
```

Comment:
I'm not a fan of this pattern in a React app but as I understand this will only be temporary.

Reply:
Agreed, not a big fan of that as well ;)